### ISSUE TITLE
Make helm `--create-namespace` Parameter Optional

### ISSUE THREAD

**Issue by stolendog** [bitovi/github-actions-deploy-eks-helm#54]
The Helm flag `--create-namespace` currently does not check if the namespace exists but instead attempts to create it. This action requires cluster permissions for the namespace creation API ([reference](https://github.com/helm/helm/blob/main/pkg/action/install.go#L379)).

In my case, I only want to grant permissions on github action to the namespace itself, not for the cluster-wide namespace creation API. Could you please make the --create-namespace parameter optional?

**Comment by mcascone** [comment]
HI @stolendog! I'm looking into this now and will get back to you asap. Thanks for contributing!

**Comment by mcascone** [comment]
Looks like the issue is here:
https://github.com/bitovi/github-actions-deploy-eks-helm/blob/016d790e0f780d468a99a4ded0d10636d94daad5/deploy.sh#L126-L132

I'll create a conditional on that parameter. @LeoDiazL

**Comment by mcascone** [comment]
Hi @stolendog, i've got the change on a branch in PR #55. I haven't had time to test it out yet; if you want, you can test it by using the branch in your workflow:

```yaml
name: Deploy Helm
uses: bitovi/github-actions-deploy-eks-helm@54-namespace-create-option
```

**Comment by stolendog** [comment]
Thanks, I'm already using it. It works perfect.

**Comment by mcascone** [comment]
> Thanks, I'm already using it. It works perfect.

That's fantastic, thanks for letting us know! I've merged the PR into `main` as `v1.2.10`. I haven't deleted the branch yet so as to not disrupt your work.

Thank you for your contribution and use of our Action!

**Comment by stolendog** [comment]
thanks, you can delete the branch
